src/org/parosproxy/paros/extension/manualrequest/http/impl/HttpPanelSender.java

```diff
@@ -311,4 +322,17 @@ public URI getInvalidRedirection() {
             return invalidRedirection;
         }
     }
+    
+    private void setUseTrackingSessionState(boolean shouldUseTrackingSessionState) {
+        if (!shouldUseTrackingSessionState) {
+            if (delegate != null) {
+                delegate.setUseGlobalState(shouldUseTrackingSessionState);
+            }
+        }
+        getButtonUseTrackingSessionState().setSelected(shouldUseTrackingSessionState);
+    }
+    
+    public void setButtonTrackingSessionStateEnabled(boolean enabled) {
+        getButtonUseTrackingSessionState().setEnabled(enabled);

```

Comment:
If `enabled` is `false` we should call `setUseTrackingSessionState` (with `false`).